tests/release/release.go

```diff
@@ -74,40 +73,41 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 
 	var _ = Describe("Creation of the 'Happy path' resources", func() {
 
-		It("Create an Snapshot in dev namespace.", func() {
+		It("creates a Snapshot in dev namespace.", func(ctx SpecContext) {
 			_, err := framework.ReleaseController.CreateSnapshot(snapshotName, devNamespace, applicationName, snapshotComponents)
 			Expect(err).NotTo(HaveOccurred())
-		})
+			// We add the namespace creation timeout as this is the first test so must also take into account the code in beforeall
```

Comment:
Can you make it one less tab so it aligns with line 80? And `s/beforeall/BeforeAll/` please

Reply:
Thanks, I updated however the auto formatting of my VSCode IDE returns the comments to same tab place (it believes that is the correct placement)